.ci/travis_script.sh

```diff
@@ -7,11 +7,13 @@ if [[ -n ${TOXENV} ]]; then
 elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb && \
     -z "${BIGCHAINDB_DATABASE_SSL}" ]]; then
     # Run the full suite of tests for MongoDB over an unsecure connection
-  pytest -sv --database-backend=mongodb --cov=bigchaindb
+  pytest -sv --database-backend=mongodb -m "serial"
+  pytest -sv --database-backend=mongodb --cov=bigchaindb -m "not serial"
 elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb && \
     "${BIGCHAINDB_DATABASE_SSL}" == true ]]; then
     # Run a sub-set of tests over SSL; those marked as 'pytest.mark.bdb_ssl'.
   pytest -sv --database-backend=mongodb-ssl --cov=bigchaindb -m bdb_ssl
 else
-  pytest -sv -n auto --cov=bigchaindb
+  pytest -sv -m "serial"
+  pytest -sv --cov=bigchaindb -m "not serial"
```

Comment:
Why only have `--cov=bigchaindb` for the "not serial" tests?

This block might be easier to understand if you added a comment like:
```text
# Run the full suite of tests for RethinkDB (the default backend when testing)
```

Reply:
I will add the comment and code coverage needs to be checked once, that is my understanding, please correct me if I am wrong. Otherwise I can add `--cov=bigchaindb` in the `serial` marked tests as well.